### ISSUE TITLE
Use 1024*1024 image still error:  The size of tensor a (3072) must match the size of tensor b (256) at non-singleton dimension 1

### ISSUE THREAD

**Issue by xueqing0622** [TheMistoAI/MistoControlNet-Flux-dev#7]
!!! Exception during processing !!! The size of tensor a (3072) must match the size of tensor b (256) at non-singleton dimension 1
Traceback (most recent call last):
  File "F:\ComfyUI\ComfyUI\execution.py", line 317, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\ComfyUI\ComfyUI\execution.py", line 192, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\ComfyUI\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "F:\ComfyUI\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\ComfyUI\ComfyUI\custom_nodes\MistoControlNet-Flux-dev\nodes.py", line 192, in sampling
    x = denoise_controlnet(
        ^^^^^^^^^^^^^^^^^^^
  File "F:\ComfyUI\ComfyUI\custom_nodes\MistoControlNet-Flux-dev\modules\utils.py", line 429, in denoise_controlnet
    pred = forward_mistoCN(
           ^^^^^^^^^^^^^^^^
  File "F:\ComfyUI\ComfyUI\custom_nodes\MistoControlNet-Flux-dev\modules\utils.py", line 358, in forward_mistoCN
    vec = vec + model.guidance_in(timestep_embedding(guidance, 256))
          ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
RuntimeError: The size of tensor a (3072) must match the size of tensor b (256) at non-singleton dimension 1

**Comment by baizh0u** [comment]
use git pull to upgrade this repo can fixed this.

**Comment by xueqing0622** [comment]
yes, fixed.

**Comment by George0726** [comment]
I have updated my was-node-suite-comfyui and MistoControlNet-Flux-dev. However, I am still confronting this problem.

```
ComfyUI:   c37485514b9b88934078b372ebcd62c7480c41d8 a178e25 Fix Flux FP64 math on XPU (#4210)
* master                                   00a5d08 [origin/master] Lower fp8 lora memory usage.
MistoControlNet-Flux-dev: * main 79e3982 [origin/main] Update nodes.py
was-node-suite-comfyui: * main fdb0f52 [origin/main] Merge pull request #465 from drozbay/dev-pr
```
